lib/eblox/data/post.ex

```diff
@@ -19,6 +19,40 @@ defmodule Eblox.Data.PostFSM do
 
   use Finitomata, @fsm
 
+  defmodule Tag do
+    @moduledoc false
+    @behaviour Md.Transforms
+
+    @href "/tags/"
+
+    @impl Md.Transforms
+    def apply(md, text) do
+      {:a, %{class: "tag", href: @href <> text}, [md <> text]}
+    end
+  end
+
+  defmodule EbloxParser do
+    @moduledoc false
+    @eblox_syntax Md.Parser.Syntax.merge(
+                    Application.compile_env(:eblox_syntax, :md_syntax, %{
+                      magnet: [
+                        {"🏷️", %{transform: Tag, terminators: [], greedy: false}}
+                      ]
+                    })
+                  )
+
+    use Md.Parser, syntax: @eblox_syntax
+  end
+
+  defmodule Walker do
+    @moduledoc false
+    def prewalk({:a, %{class: "tag"}, text} = elem, acc),
+      do: {elem, Map.update(acc, :tags, [text], &[text | &1])}
```

Comment:
Collecting tags.

Reply:
What is the accumulator here? The result of the prewalk, i.e. some data we want to collect from the XML tree, right?

Reply:
Yes, right. We start in line 68 with an empty map (default acc for [`XmlBuilder.prewalk/2`](https://hexdocs.pm/xml_builder_ex/XmlBuilder.html#prewalk/2),) and end up with all the info we want to collect.

Reply:
I noticed that `text` is list here. Is there any reason for it to be a list?

Reply:
> I noticed that `text` is list here. Is there any reason for it to be a list?

`text` is a `binary()`, `tags` in `properties` is indeed a list because one post can have many tags.